web/src/modules/topic/components/Score/ScorePie.tsx

```diff
@@ -30,28 +43,33 @@ export const ScorePie = ({ pieDiameter, arguableId, arguableType }: Props) => {
   });
 
   return (
-    <Box display="flex" width={pieDiameter} height={pieDiameter}>
+    <StyledBox display="flex" width={pieDiameter} height={pieDiameter}>
       <PieChart
         data={data}
-        radius={50} // is relative to viewbox size [100, 100]
+        radius={45} // is relative to viewbox size [100, 100]
         label={({ dataEntry }) => dataEntry.key}
         labelStyle={() => ({
           fontSize: "14", // is relative to viewbox size [100, 100]
           pointerEvents: "none",
+          webKitUserSelect: "none" /* Safari */,
+          msUserSelect: "none" /* IE 10 and IE 11 */,
+          userSelect: "none" /* Standard syntax */,
         })}
         segmentsStyle={{
+          transformOrigin: "center",
           transition: theme.transitions.create("all", {
             duration: theme.transitions.duration.shortest,
           }),
+          pointerEvents: "visiblePainted",
```

Comment:
do you know if `visiblePainted` does anything special over `auto`? from a brief glance https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events it looks like they should behave the same for SVGs - and if that's the case, I'd just use `auto` here since that's equivalent to having no pointer-events property specified

Reply:
`pointerEvents: "auto"` will work just fine.